rfc/rfc006-distributed-registry.md

```diff
@@ -279,6 +279,16 @@ When control over a DID document has been lost, the DID subject will have to hav
 Without Verifiable Credentials linked to the DID document, the DID document no longer has any value.
 The DID subject will have to go through the process of reacquiring all Verifiable Credentials for a new DID document.
 
+#### 3.3.4 Protection against public key zerging
+
+By registering a very large number of public keys in a DID Document an attacker can exhaust resources used for storing public keys.
+
+To prevent this from happening a DID Document MUST contain no more than 50 keys.
+
+TODO: What if the attacker just creates 100.000 versions of a document with in each 50 new keys?
```

Comment:
That would require 100.00 transactions which are noticeable and can be prevented.

Reply:
Agreed, that should be an identified threat in RFC004 (transaction flooding)